db/migrate/20210128094853_add_work_history_status_enum.rb

```diff
@@ -0,0 +1,13 @@
+class AddWorkHistoryStatusEnum < ActiveRecord::Migration[6.0]
+  def up
+    remove_column :application_forms, :work_history_status, :string
+    create_enum "work_history_type", %w[can_complete full_time_education can_not_complete]
```

Comment:
I’d be tempted to name the enum `work_history_status` as it’s not really a type? Or alternatively something reflecting the UI?

```suggestion
    create_enum "work_history_answer", %w[yes no_full_time_education no_work_history]
```

Reply:
Yea that makes sense. I'll make that change, but hold off on thinking about merging until 👇 that discussion is had.